src/VueGAPI/GoogleAuthService.js

```diff
@@ -5,16 +5,34 @@ export default class GoogleAuthService {
     this.authInstance = null
 
     this.login = this.login.bind(this)
+    this.refreshToken = this.refreshToken.bind(this)
     this.setSession = this.setSession.bind(this)
     this.logout = this.logout.bind(this)
     this.isAuthenticated = this.isAuthenticated.bind(this)
   }
 
+  // NOTE: handle expiresAt method, this is private
+  // Set the time that the access token will expire at
+  _expiresAt = (authResult) => {
+    return JSON.stringify(authResult.expires_in * 1000 + new Date().getTime())
+  }
+
   login (event) {
     return this.authInstance.signIn()
       .then(this.setSession)
   }
 
+  refreshToken (event) {
+  console.log('REFRESHED TOKEN')
```

Comment:
@	hkd987 you might want to remove this.

Reply:
@hkd987 do you still need this log yet? Else let's remove it before the merge :)
```diff
-  console.log('REFRESHED TOKEN')
```

Reply:
I let it there to be similar to what the style you had going where you are logging when GAPI is installed. We can remove it if you want.